src/lib/operators/validate.cpp

```diff
@@ -74,6 +86,23 @@ std::shared_ptr<const Table> Validate::_on_execute(std::shared_ptr<TransactionCo
   auto job_end_chunk_id = ChunkID{0};
   auto job_row_count = uint32_t{0};
 
+  // In some cases, we can identify a chunk as being entirely visible for the current transaction. Simply said,
+  // if the youngest row in a chunk is visible, all other rows are older and hence visible, too. This
+  // (1) all rows in the chunk have been commited (i.e., their begin_cid has been set),
+  // (2) the highest begin_cid in the chunk is lower/equal than the snapshot_cid of the transaction
```

Comment:
(1) lower than / equal to
(2) meaning that all rows have been inserted and committed before this transaction was started